modules/varnish/templates/default.vcl.erb

```diff
@@ -79,9 +79,21 @@ sub vcl_recv {
   # Strip cookies from inbound requests. Corresponding rule in vcl_fetch{}
   # For simplicity and security most applications should not use cookies.
   # With the exception of:
-  #   - Licensing
+  #   - Licensing application
+  #   - AB Testing cookies in any application
   if (req.url !~ "^/apply-for-a-licence") {
-    unset req.http.Cookie;
+    if (req.http.Cookie) {
+      set req.http.Cookie = ";" + req.http.Cookie;
+      set req.http.Cookie = regsuball(req.http.Cookie, "\s", "");
+      # TODO: Read from a cookie whitelist in a config file, rather than hard-coding allowed cookies
```

Comment:
I've left this TODO in here because I'd like to test the simple, hard-coded version on Integration before adding a whitelist. If that's not appropriate, I can take it out and add it to a checklist on Trello.